src/dataflow/src/server.rs

```diff
@@ -59,8 +62,17 @@ mod metrics;
 /// back to the coordinator.
 static TS_BINDING_FEEDBACK_INTERVAL_MS: u128 = 1_000;
 
+// A dirty hack to allow us to derive EnumKind:
```

Comment:
I don't quite follow what this comment is referring to, which part is the dirty hack?

Reply:
Oh wow, oops - I had moved that enum definition into a private module in the hope that this would circumvent the missing_docs forbid; but then forgot to remove the comment. Willfix!